### DISCUSSION TITLE
Would be cool if it supported Bluesky

### DISCUSSION THREAD

**Discussion by RoyTheunissen** [badges/shields#10671] [Ideas] [answered]
Bluesky's really picking up some momentum as an alternative to X, might be cool if there were a badge for it, even just a quick static one just to be able to cover it at all.

**Comment by chris48s** [comment, marked as answer]
If you just want a static badge you can do something like

[![Follow @roytheunissen.bsky.social](https://img.shields.io/badge/follow-@roytheunissen.bsky.social-whitesmoke?style=social&logo=bluesky)](https://bsky.app/profile/roytheunissen.bsky.social)

```
[![Follow @roytheunissen.bsky.social](https://img.shields.io/badge/follow-@roytheunissen.bsky.social-whitesmoke?style=social&logo=bluesky)](https://bsky.app/profile/roytheunissen.bsky.social)
```

using features that already exist for that purpose.

If bluesky exposes a public API that would allow us to get some useful data point like a number of followers, feel free to raise a [badge request](https://github.com/badges/shields/issues/new?assignees=&labels=service-badge&projects=&template=3_Badge_request.yml) with the details

**Reply by voxpelli** [reply]
For reference for those who finds this discussion: There is such a badge request now: https://github.com/badges/shields/issues/10788
